src/filesystem/impls/filer/BracketsFiler.js

```diff
@@ -3,16 +3,107 @@
 define(function (require, exports, module) {
     "use strict";
 
-    // If you need to debug Filer for some reason, drop the .min below
+    // TODO: we shouldn't be loading this just to get Path, Buffer, etc.
```

Comment:
Somehow we need to just do:
```js
var Path = require("thirdparty/filer/dist/path");
var Buffer = require("thirdparty/filer/dist/buffer");
```
So that we don't have to load all the rest of Filer, which we won't use at all.  This is about download size savings.